src/lib/services/access-service.ts

```diff
@@ -154,21 +214,29 @@ export class AccessService {
         return userRoles.filter((r) => r.type === RoleType.ROOT);
     }
 
-    async removeUserFromRole(userId: number, roleId: number): Promise<void> {
-        return this.store.removeUserFromRole(userId, roleId);
+    async removeUserFromRole(
+        userId: number,
+        roleId: number,
+        projectId: string,
+    ): Promise<void> {
+        return this.store.removeUserFromRole(userId, roleId, projectId);
     }
 
     async addPermissionToRole(
         roleId: number,
         permission: string,
-        projectId?: string,
+        environment?: string,
```

Comment:
Does this change have any side effects? Also noticing that the function below `removePermissionFromRole` still has projectId as a parameter. I would imagine that too needs to change? Probably just semantics in terms of parameters, but best to be explicit.

Reply:
Oddly enough, no, since the only place it's ever called is in the tests. But that change looks dangerous as hell, agreed, so I've added a comment above this (and it's equal and opposite removePermissionFromRole) to point out that they exist for testing reasons